operate/constants.py

```diff
@@ -52,3 +52,4 @@
 MECH_ACTIVITY_CHECKER_JSON_URL = "https://raw.githubusercontent.com/valory-xyz/autonolas-staking-programmes/refs/heads/main/abis/0.8.25/SingleMechActivityChecker.json"
 SERVICE_REGISTRY_TOKEN_UTILITY_JSON_URL = "https://raw.githubusercontent.com/valory-xyz/open-autonomy/refs/tags/v0.18.4/packages/valory/contracts/service_registry_token_utility/build/ServiceRegistryTokenUtility.json"  # nosec
 MECH_AGENT_FACTORY_JSON_URL = "https://raw.githubusercontent.com/valory-xyz/ai-registry-mech/main/abis/0.8.25/AgentFactory.json"
+MECH_MARKETPLACE_JSON_URL = "https://raw.githubusercontent.com/valory-xyz/mech-quickstart/refs/heads/main/contracts/MechMarketplace.json"
```

Comment:
because the mech-quickstart main branch may change it
```suggestion
MECH_MARKETPLACE_JSON_URL = "https://raw.githubusercontent.com/valory-xyz/ai-registry-mech/refs/tags/v0.4.0-pre-internal-audit4/abis/0.8.28/MechMarketplace.json"
```